Source/CsDebugScript.Engine/Engine/Utility/DictionaryCache.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections;
```

Comment:
Yep - IEnumerable is in it.

---
In reply to: [70602281](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r70602281) [](ancestors = 70602281)